examples/lusid/quotes/Upload_ECB_FX_Quotes.sql

```diff
@@ -0,0 +1,34 @@
+-- =====================================================
+-- Description:
+-- In this query, we pull FX Spot data from the ECB Provider and upsert into Lusid
+-- This can be run in query editor or used in a schedule
+-- This can be used to upload all data from 1999-01-01 by changing start_date
+-- Alternatively it can be used in a schdule to update FX values daily
+-- Note FX rates are uploaded to ECB between 4-5pm GMT
+-- =====================================================
+@@start_date = select date('now'); --Gets todays date, alternatively provide your start date
```

Comment:
This causes the query to have zero rows.

```suggestion
@@start_date = select date('now', '-1 days'); --Gets todays date, alternatively provide your start date
```